config/checkstyle/checkstyle.xml

```diff
@@ -68,7 +68,6 @@
         <!-- Checks for Javadoc comments.                     -->
         <!-- See http://checkstyle.sf.net/config_javadoc.html -->
         <module name="JavadocMethod">
-            <property name="scope" value="public"/>
```

Comment:
This change was necessary for gradle 8.0 compatibility. The scope trait was removed in the newer checkstyle plugin